Assets/MRTK/Examples/Experimental/SceneUnderstanding/Profiles/DemoSUProfile.asset

```diff
@@ -20,4 +20,4 @@ MonoBehaviour:
     componentName: Windows Scene Understanding Observer
     priority: 0
     runtimePlatform: 8
-    observerProfile: {fileID: 11400000, guid: c16ec84a27c8162459bd27e1758712bd, type: 2}
+    observerProfile: {fileID: 0}
```

Comment:
Lost profile reference here

Reply:
weird! wonder how....

i may start over on this change